aws/values.yaml

```diff
@@ -1,17 +1,55 @@
 proxy:
+  https:
+    enabled: false
   service:
     annotations:
       service.beta.kubernetes.io/aws-load-balancer-type: nlb
 
 scheduling:
   userScheduler:
     enabled: false
+  userPods:
+    nodeAffinity:
+      matchNodePurpose: require
 
 prePuller:
   continuous:
     enabled: false
 
 singleuser:
+  defaultUrl: "/lab"
+  startTimeout: 600
+  cpu:
+    limit: 4
+    guarantee: 3
+  memory:
+    limit: 16G
+    guarantee: 14G
+
+  profileList:
+    - display_name: "Base-notebook (m5.2xlarge | 4 cores, 16Gb)"
+      description: "latest base-notebook from pangeo-stacks"
+      kubespawner_override:
+        image: pangeo/base-notebook:2019.12.07
+    - display_name: "Pangeo-notebook (m5.2xlarge | 4 cores, 16Gb)"
+      description: "latest pangeo-notebook from pangeo-stacks"
+      default: true
+      kubespawner_override:
+        image: pangeo/pangeo-notebook:2019.12.07
+
   storage:
     dynamic:
-      storageClass: aws-efs
\ No newline at end of file
+      storageClass: aws-efs
+
+hub:
+  resources: # should fill up single node to near capacity
```

Comment:
I think the hub needs far less resources, we should definitely not take up a full single node with it.

This is what I see on one of our largest hubs:

```
    resources:
      limits:
        memory: 1Gi
      requests:
        cpu: 200m
        memory: 256Mi
```